src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -577,16 +553,15 @@
     short-summary: The deployment mode.
 examples:
   - name: Validate whether a template is valid at resource group.
-    text: az deployment group validate --resource-group testrg --template-file {template-file}
-  - name: Validate whether a template is valid at resource group. (autogenerated)
-    text: |
-        az deployment group validate --parameters MyValue=This MyArray=@array.json --resource-group testrg --template-file azuredeploy.json
-    crafted: true
+    text: az deployment group validate --resource-group MyResourceGroup --template-file {template-file}
+  - name: Validate whether a template is valid at resource group.
+    text: >
+        az deployment group validate --parameters MyValue=This MyArray=@array.json --resource-group MyResourceGroup --template-file azuredeploy.json
 """
 
 helps['deployment group create'] = """
 type: command
-short-summary: Start a deployment at resource group.
+short-summary: Create a deployment group.
```

Comment:
In fact, the feature of this command is to create a resource group level deployment, so the previous description`Start a deployment` may be more appropriate.
```suggestion
short-summary: Start a deployment at resource group scope.
```